tests/test_html_rewriting.py

```diff
@@ -0,0 +1,174 @@
+import pytest
+from warc2zim.content_rewriting import rewrite_html
+from textwrap import dedent
+
+
+@pytest.fixture(
+    params=[
+        "A simple string without url",
+        "<html><body><p>This is a sentence with a http://exemple.com/path link</p></body></html>",
+        '<a data-source="http://exemple.com/path">A link we should not rewrite</a>',
+        'p style="background: url(some/image.png)">A link in a inline style</p>',
```

Comment:
You changed the label to `A url (relative) in a inline style` but it's still misleading. It looks like this should be rewrote but the fact that it's not has nothing to do with it being relative of not but the fact that it's not an actual inline style attr because this is outside a node.

Reply:
Haaa, but it should be in a node. It missing a `<` at the beginning of the string.